libcnb-test/src/pack.rs

```diff
@@ -213,6 +213,64 @@ pub(crate) fn run_pack_command<C: Into<Command>>(
     output
 }
 
+#[derive(Clone, Debug)]
+pub(crate) enum BuildpackPackageFormat {
+    Image,
+}
+
+#[derive(Clone, Debug)]
+pub(crate) struct PackBuildpackPackageCommand {
+    name: String,
+    config: PathBuf,
+    format: BuildpackPackageFormat,
+}
+
+impl PackBuildpackPackageCommand {
+    pub(crate) fn new(name: impl Into<String>, config: impl Into<PathBuf>) -> Self {
+        Self {
+            name: name.into(),
+            config: config.into(),
+            format: BuildpackPackageFormat::Image,
+        }
+    }
+}
+
+impl From<PackBuildpackPackageCommand> for Command {
+    fn from(value: PackBuildpackPackageCommand) -> Self {
+        let mut command = Self::new("pack");
+
+        let args = vec![
+            String::from("buildpack"),
+            String::from("package"),
+            value.name,
+            String::from("--config"),
+            value.config.to_string_lossy().to_string(),
+            format!(
+                "--format={}",
+                match value.format {
+                    BuildpackPackageFormat::Image => String::from("image"),
+                }
+            ),
+        ];
+
+        command.args(args);
+
+        command
+    }
+}
+
+pub(crate) fn run_buildpack_package_command<C: Into<Command>>(command: C) {
```

Comment:
(there's the new `run_command` util after #619 btw; though moot point given this can be removed due to the Pack CLI bug being fixed)